source/tools/hubspot.rst

```diff
@@ -0,0 +1,111 @@
+Hubspot
+============
+
+What is Hubspot?
+----------------
+HubSpot is an "inbound marketing software platform" which offers features for internet marketing, such as emails, blogging and social media. This piece of software is mostly used by the marketing team.
+
+This document will mostly cover tips on how to use the emailer tool/rehabstudio email template.
+
+The rehabstudio email template
+------------------------------
+
+The 2 in 1 template allows you to switch between the event or newsletter email layouts. Each layout includes a spotlight, monthly links and footer section. The spotlight and monthly links can be hidden if not needed.
```

Comment:
@KirMcC maybe add just a line before this explaining what the Event and Newsletter emails actually are? Might give a bit of context for people coming to this blind.